app/components/review_gcse_qualification_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<section class="app-summary-card govuk-!-margin-bottom-6">
+  <header class="app-summary-card__header">
+    <h3 class="app-summary-card__title">
+      GCSE
+    </h3>
+  </header>
+  <div class="app-summary-card__body">
+    <dl class="govuk-summary-list">
+      <% gcse_qualification_rows.each do |row| %>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
+            <%= row[:key] %>
+          </dt>
+          <dd class="govuk-summary-list__value">
+            <%= row[:value] %>
+          </dd>
+          <dd class="govuk-summary-list__actions">
+            <%= link_to 'Change', 'edit_qualification_path' %>
```

Comment:
As a heads up, you can use the view helper `govuk_link_to` which adds the `govuk-link` CSS class. Not need to do anything here though when you refactor to use `SummaryCardComponent` in your GCSE review component.